cfme/containers/pod.py

```diff
@@ -8,6 +8,12 @@
 
 list_tbl = CheckboxTable(table_locator="//div[@id='list_grid']//table")
 
+
```

Comment:
A pod is related to a provider, I really think you want this against a provider, and we need to make sure you get to the pods screen through the provider else if you have more than one containers provider added, your list of pods will be longer than it should be